src/wasm-interpreter.h

```diff
@@ -1898,6 +1899,23 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
         }
         return makeGCData(contents, curr->type);
       }
+      case StringNewFromCodePoint: {
+        uint32_t codePoint = ptr.getSingleValue().geti32();
+        if (codePoint > 0x10FFFF) {
+          trap("invalid code point");
+        }
+        std::stringstream wtf16;
+        String::writeWTF16CodePoint(wtf16, codePoint);
+        std::string str = wtf16.str();
+        Literals contents;
+        // The output contains a code point and a null terminator, so it is not
+        // empty. Write it all out but the null terminator.
+        assert(!str.empty());
+        for (size_t i = 0; i < str.size() - 1; i++) {
+          contents.push_back(Literal(int32_t(str[i])));
```

Comment:
For this part you need to consume two bytes for each entry in the array. The `Literal(std::string_view)` constructor can do this for you, though.

Reply:
Thanks, done.